packages/server/postgres/migrations/1696275124427_samlMetadataURL.ts

```diff
@@ -0,0 +1,21 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+  ALTER TABLE "SAML"
+  ADD COLUMN IF NOT EXISTS "metadataURL" VARCHAR(2048);
```

Comment:
microsoft azure only allows urls of 2048 chars, so borrowing their limit here
we could still run into problems since this URL gets encoded into another URL, but under normal usage we shouldn't have a URL that is longer than 200 chars.